tests/src/test/java/alluxio/client/RemoteBlockInStreamIntegrationTest.java

```diff
@@ -86,6 +86,7 @@ public RemoteBlockInStreamIntegrationTest(String dataServer, String transferType
     mLocalAlluxioClusterResource = new LocalAlluxioClusterResource.Builder()
         .setProperty(PropertyKey.WORKER_DATA_SERVER_CLASS, dataServer)
         .setProperty(PropertyKey.WORKER_NETWORK_NETTY_FILE_TRANSFER_TYPE, transferType)
+        .setProperty(PropertyKey.USER_BLOCK_REMOTE_READER_CLASS, reader)
```

Comment:
I think we can remove this change and the next change , right?
just to keep them in doc and in supported keys. not necessary to reference them